src/Microsoft.ML.DataView/Microsoft.ML.DataView.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
+    <PackageReference Include="System.Drawing.Common" Version="$(SystemDrawingCommonPackageVersion)" />
```

Comment:
If we really do go through with this (I hope we can do something so we don't have to), you will need to update the PackageReference in the corresponding `nupkgproj` under the `pkg` folder as well.